vision/product-search/cloud-client/src/main/java/com/example/vision/ProductInProductSetManagement.java

```diff
@@ -135,6 +144,50 @@ public static void removeProductFromProductSet(
   }
   // [END vision_product_search_remove_product_from_product_set]
 
+  // [START vision_product_search_purge_products_in_product_set]
```

Comment:
For all new samples, we're asking people to follow the new guidelines:

Key thing here is really: "Each snippet should be be contained in its own file"

https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md

Examples: https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/dlp/src/main/java/dlp/snippets

Reply:
Okay I will create separate its own file.

Reply:
Should I deconstruct all methods to keep everything consistent? Or Is it only for purge related snippets?

Reply:
Um mostly up to you. We decided to only ask for newly created stuff to follow the guidelines or when doing an overhaul to the existing samples. 

So you could leave them as is.

Reply:
Okay I did it, please provide a feedback on new formatting